server/task_executor_pitr_restore.go

```diff
@@ -103,6 +104,7 @@ func (exec *PITRRestoreTaskExecutor) doPITRRestore(ctx context.Context, task *ap
 			zap.Error(err))
 		return fmt.Errorf("failed to get latest backup before or equal to targetTs[%d], error[%w]", targetTs, err)
 	}
+	log.Debug("Got latest backup before or equal to targetTs", zap.String("backup", backup.Name))
```

Comment:
this log is very similar to L99

Reply:
Yes, it's "Get" and "Got", namely before and after.
Any suggestions?